docs/080-breaking-changes.rst

```diff
@@ -19,6 +19,17 @@ the compiler notifying you about it.
   Checks for overflow are very common, so we made them the default to increase readability of code,
   even if it comes at a slight increase of gas costs.
 
+* ABI coder v2 is activated by default.
+
+  You can choose to use the old behaviour using ``pragma abicoder v1;``.
+  The pragma ``pragma experimental ABIEncoderV2;`` is still valid, but it is deprecated and has no effect. If you want to...
+  but if you want to be explicit, please use ``pragma abicoder v2;`` instead.
```

Comment:
```suggestion
  The pragma ``pragma experimental ABIEncoderV2;`` is still valid, but it is deprecated and has no effect.
  If you want to be explicit, please use ``pragma abicoder v2;`` instead.
```

This time a proper two-line suggestion without ellipses, s.t. it's fine to directly apply ;-). Sorry for not doing that before.

Reply:
sorry, should have paid more attention.